packages/prisma/schema.prisma

```diff
@@ -775,10 +775,11 @@ model ApiKey {
 }
 
 model HashedLink {
-  id          Int       @id @default(autoincrement())
-  link        String    @unique()
-  eventType   EventType @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
-  eventTypeId Int       @unique
+  id           Int       @id @default(autoincrement())
+  link         String    @unique()
+  eventType    EventType @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
+  eventTypeId  Int
+  destroyOnUse Boolean   @default(false)
```

Comment:
The feature defines a single-use link. Additionally, the user does not have the ability to set the `destroyOnUse` value, so the `destroyOnUse` field is unnecessary and could always be set to true for this feature